zerver/lib/actions.py

```diff
@@ -3912,26 +3912,21 @@ def do_send_confirmation_email(invitee, referrer, body):
     send_email('zerver/emails/invitation', to_email=invitee.email, from_name=from_name,
                from_address=FromAddress.NOREPLY, context=context)
 
-def user_email_is_unique(email):
-    # type: (Text) -> None
-    try:
-        get_user_profile_by_email(email)
-        raise ValidationError('%s already has an account' % (email,))
-    except UserProfile.DoesNotExist:
-        pass
+def email_not_system_bot(email: Text) -> None:
+    if email.lower() in settings.CROSS_REALM_BOT_EMAILS:
```

Comment:
There's some redundancy between different code using `settings.CROSS_REALM_BOT_EMAILS` directly vs. `zulip.models.get_cross_realm_emails`, which would be nice to clean up. Can be a small followup after this branch, though.

Reply:
Yeah, we wanted to be particularly careful here with case bugs; so we should probably add a test for trying to slip by with different case, and then refactor away.